wavefront-istio-adapter/values.yaml

```diff
@@ -0,0 +1,25 @@
+adapter:
+  image: vmware/wavefront-adapter-for-istio
```

Comment:
For consistency with our other charts, these should just be top level properties similar to the prom storage adapter / HPA adapter.